lib/Mojolicious/Sessions.pm

```diff
@@ -140,6 +142,38 @@ A callback used to deserialize sessions, defaults to L<Mojo::JSON/"j">.
     return {};
   });
 
+=head2 samesite
+
+  my $str   = $sessions->samesite;
+  $sessions = $sessions->samesite('Lax')
+
+SameSite attribute for session cookies.
+
+This attribute is used to control when the browser is allowed to send cookies
+for cross-site requests.
+
+=over 4
+
+=item C<"Lax">
+
+This is the default value.
+
+Cookies will be sent for clicking on links and for C<method="get"> forms. They
+will not be sent for cross-site images, iframes, ajax, and C<method="post">
+forms.
+
+=item C<"Strict">
+
+Cookies will never be sent for cross-site requests. Even when the user follows a
+link to the application, the session cookies will not be sent.
+
+=item C<undef>
+
+Cookies will be sent for all cross-site requests. This is the traditional
+behavior of cookies.
+
+=back
```

Comment:
We normally link to the specs in cases like this, instead of trying to write our own explanations.

Reply:
Ok. I was wondering if this was the case. Though it's N-level deep in the spec to understand how this impacts experience.

Reply:
`Mojo::Headers` for example has a lot of these. https://mojolicious.org/perldoc/Mojo/Headers#origin